Tools/SeeDot/seedot/compiler/ir/irBuilder.py

```diff
@@ -1513,7 +1513,7 @@ def visitFunc(self, node: AST.Func):
         elif node.op == SeeDotParser.SGN:
             return self.visitSgn(node)
         elif node.op == SeeDotParser.TANH:
-            if useNewTableExp() and forFixed():
+            if useNewTableExp() and forFixed() and self.vbwEnabled:
```

Comment:
useNewTableExp() pe ek assert daal do. useNewTableExp() can only be true if self.vbwEnabled is true. User should change config values.

Reply:
adding this AND will make debugging harder

Reply:
Is it not a necessary condition to use this table?  

it already has an assert inside. But the assertion fails and error is not shown because of try/catch block.